### ISSUE TITLE
Amperfy crashes when opened in CarPlay (iOS 15.4.1)

### ISSUE THREAD

**Issue by Jspeo** [BLeeEZ/amperfy#309]
Amperfy crashes when opened in CarPlay (iOS 15.4.1)

**Comment by BLeeEZ** [comment]
Is this still occurring in v1.2.2 (2)?

**Comment by Jspeo** [comment]
> Is this still occurring in v1.2.2 (2)?

Yes, there will still be crashes in v1.2.2 (3) at present.

**Comment by BLeeEZ** [comment]
I have a question @Jspeo: Do you have other CarPlay Apps installed on your device? I found out that the Tab view I user in CarPlay has a maximum number of tabs. This maximum depends on the car manufacturer (https://developer.apple.com/documentation/carplay/cptabbartemplate). Could you please tell me how many tabs other apps  in your car are displaying?

**Comment by BLeeEZ** [comment]
Amperfy currently uses 4 tabs. I added a check for maximumTabCount (20fecc57a4f6f10e5d143d40feb861cd0532c3de). This will reduce the number of used tabs accordingly.

**Comment by Jspeo** [comment]
> I have a question [@Jspeo](https://github.com/Jspeo): Do you have other CarPlay Apps installed on your device? I found out that the Tab view I user in CarPlay has a maximum number of tabs. This maximum depends on the car manufacturer (https://developer.apple.com/documentation/carplay/cptabbartemplate). Could you please tell me how many tabs other apps in your car are displaying?

Yes,i have other carplay apps installed on my device ,about 10 tabs
